docs/contribute/guidelines.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Coding and Commit Guidelines
```

Comment:
This list looks _much_ shorter than the original one. Which sections did you remove and why?

Reply:
This change does not contain a full documentation. Therefore the guideline page does not contain all information.

I had the following plan with the [old content](http://www.saros-project.org/guidelines):
* Ticketing - is out-dated and will be removed
* Contribution Rules- referenced in #111 (see file old/contribution.md)
* Coding Conventions - referenced in #111 (see file old/coderules.1.md)
* Usability - referenced in #111 (see file old/usability.md)
* Continuous Integration with Jenkins - out-date and will be replaced

The files references in #111 will be transferred and reworked.